setupckern.F90

```diff
@@ -319,11 +319,15 @@ subroutine setupckern(carma, cstate, rc)
                 pe3 = pe**(1._f/3._f)
 
                 ! <ccd> is convective diffusion coagulation coefficient
-                if( re_larg .lt. 1._f )then
-                  ccd = 0.45_f*cbr*pe3
-                else
-                  ccd = 0.45_f*cbr*pe3*re_larg**(ONE/6._f)
-                endif
+                if (do_aer_cld_interact) then
+     			  if( re_larg .lt. 1._f )then
+                    ccd = 0.45_f*cbr*pe3
+                  else
+                    ccd = 0.45_f*cbr*pe3*re_larg**(ONE/6._f)
+                  endif
+  				else
+     			  ccd = 0._f
+  				end if
```

Comment:
The indentation needs to be fixed.  You might have a mixture of spaces and tabs.  Please don't use tabs.  Just use spaces.